examples/diff_adv/source_heat.jl

```diff
@@ -0,0 +1,122 @@
+using Catlab
```

Comment:
Why catlab

Reply:
Needed for ```nparts```, I'm going to delete this file anyway since this was just experimental and doesn't actually work.